recipes-browser/wpewebkit/wpewebkit_2.46.0.bb

```diff
@@ -0,0 +1,15 @@
+require wpewebkit.inc
+require conf/include/devupstream.inc
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI = "https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz;name=tarball \
+           file://0001-CMake-WPE-ENABLE_WPE_PLATFORM-cannot-be-toggled.patch \
```

Comment:
You can use [WPE WebKit 2.46.1](https://wpewebkit.org/release/wpewebkit-2.46.1.html) now, and drop this patch that has been included in the release.